aeg.html

```diff
@@ -0,0 +1,28 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet"  href="style.css">
+</head>
+<body>
+    
+    <h1>Alpha Existential Graphs</h1>
+   
+    <button class="button">Cut</button>
+
+    <button class="button">Atom</button>
+
+    <div include-html="components/navbar.html"></div>
```

Comment:
Please explain what this for, what is "components/navbar.html", no such file exists.